apps/playground/src/app/navigation/supporting-pages/recursion/recursion.component.html

```diff
@@ -21,6 +21,6 @@ <h3>Demo:</h3>
     // Has filename and languageType
     $(document).ready(() => {{ '{' }}
     // do something
-    });
+    &#125;);
```

Comment:
For my knowledge - why are these changing?

Reply:
The new control flow syntax uses `@` and `}` as control characters, so Angular runs a schematic to encode them.